docs/source/Modules/Encryption.rst

```diff
@@ -3,7 +3,8 @@ Encryption (Version 1)
 
 .. note::
     External module required!!!
-    If you wish to use this module, please go and download `Cryptography`_. Otherwise this module is useless
+    If you wish to use this module, please go and download `Cryptography`_. Otherwise this module is useless.
```

Comment:
Could you also add a note for termux users about how it doesn't work (easily) without rust etc